test/Android/androidtest.cmd

```diff
@@ -14,10 +14,11 @@ if not defined VisualStudioVersion (
 
 set _DeviceId=
 set _Platform=
-set _SdkRoot=%ProgramFiles(x86)%\Android\android-sdk
-set _NdkRoot=%ProgramData%\Microsoft\AndroidNDK\android-ndk-r11c
+set _SdkRoot=c:\Microsoft\AndroidSDK\25
```

Comment:
nit: `%HOMEDRIVE%`?

Reply:
Is that always the case? I feel like they might not be.